Stubs/Xamarin.Forms.Platform.Tizen/Properties/AssemblyInfo.cs

```diff
@@ -1,31 +1,7 @@
-﻿using System.Reflection;
-using System.Runtime.CompilerServices;
+﻿using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-// General Information about an assembly is controlled through the following 
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyDescription ("")]
-[assembly: AssemblyCopyright ("Copyright ©  2018")]
-[assembly: AssemblyTrademark ("")]
-[assembly: AssemblyCulture ("")]
 [assembly: InternalsVisibleTo ("Xamarin.Forms.Core")]
-
-// Setting ComVisible to false makes the types in this assembly not visible 
-// to COM components.  If you need to access a type in this assembly from 
-// COM, set the ComVisible attribute to true on that type.
 [assembly: ComVisible (false)]
-
 // The following GUID is for the ID of the typelib if this project is exposed to COM
-[assembly: Guid ("d3f9fea6-cce8-4c30-b0d2-197b40c50757")]
-
-// Version information for an assembly consists of the following four values:
-//
-//      Major Version
-//      Minor Version 
-//      Build Number
-//      Revision
-//
-// You can specify all the values or you can default the Build and Revision Numbers 
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
+[assembly: Guid ("d3f9fea6-cce8-4c30-b0d2-197b40c50757")]
```

Comment:
check with the Tizen guys, but I don't think this will be used as ComVisible is set to false

Reply:
Yap, you are right. Can remove.